sql/server.go

```diff
@@ -139,8 +144,9 @@ func (p parameters) Arg(i int) (parser.Datum, bool) {
 		return parser.DString(t), true
 	case *string:
 		return parser.DString(*t), true
+	default:
+		panic(fmt.Sprintf("Incorrect type %T", t))
```

Comment:
I'll incorporate all the feedback and fix the commented out test tonight.

On Fri, Aug 7, 2015, 8:41 AM Peter Mattis <notifications@github.com> wrote:

> In sql/server.go
> <https://github.com/cockroachdb/cockroach/pull/2017#discussion_r36514401>:
>
> > @@ -139,8 +144,9 @@ func (p parameters) Arg(i int) (parser.Datum, bool) {
> >  		return parser.DString(t), true
> >  	case *string:
> >  		return parser.DString(*t), true
> > +	default:
> > +		panic(fmt.Sprintf("Incorrect type %T", t))
>
> s/Incorrect/unexpected/g?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2017/files#r36514401>.
>